opentech/apply/funds/models/submissions.py

```diff
@@ -130,6 +132,7 @@ def for_table(self, user):
                 reviews.submitted().values('submission').annotate(calc_recommendation=Sum('recommendation') / Count('recommendation')).values('calc_recommendation'),
                 output_field=IntegerField(),
             ),
+            role_assigned=Subquery(roles_for_review[:1].values('role__name')),
```

Comment:
Maybe we want the following? unless the actual role is important.
```
role_icon=Subquery(
                roles_for_review[:1].values('role__icon_id')
            )
```

Reply:
The requirements are a bit hazy. Do you think we want the icon and not the role name?

Reply:
I'm going to use role name rather than icon. As we discussed on slack, without bumping up our version of wagtail, we can't use the [generate_image_url()](https://docs.wagtail.io/en/v2.4/advanced_topics/images/image_serve_view.html#generating-dynamic-image-urls-in-python) method in Python that was made available after our current version.